src/main/java/seedu/tp/flashcard/FlashcardList.java

```diff
@@ -95,11 +95,12 @@ public FlashcardList addFlashcard(Flashcard flashcard) {
      * @param index the index of the flashcard to be deleted
      * @return the deleted flashcard
      */
-    public Flashcard deleteFlashcard(int index) throws InvalidFlashcardIndexException {
+    public void deleteFlashcard(int index) throws InvalidFlashcardIndexException {
         try {
             Flashcard flashcard = flashcards.remove(index);
             LOGGER.info("Deleted flashcard " + flashcard.getName() + " from list.");
-            return flashcard;
+            System.out.println("The following flashcard has been deleted.");
```

Comment:
Consider moving this part to the UI class? Maybe we can add a UI method for deletion confirmation and call that in the `DeleteCommand`'s `execute()` method